components/k8s-model-server/request-logger/logging_worker.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+# Copyright 2018 The Kubeflow Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Worker for request logging.
+It should be deploy as a sidecar with the serving component.
+"""
+from __future__ import print_function
+
+import argparse
+import ast
+import json
+import os
+import time
+
+from google.cloud import bigquery
```

Comment:
Why do we need to write our own logging agent as opposed to using fluentd?
For example, https://github.com/kaizenplatform/fluent-plugin-bigquery